public/robots.txt

```diff
@@ -2,6 +2,8 @@
 #
 # To ban all spiders from the entire site uncomment the next two lines:
  User-agent: *
- Disallow: /flags
+ Disallow: /admins
+ Disallow: /blockeds
+ Disallow: /activities
 
  Sitemap: http://www.catforum.mntc.in/sitemap.xml
```

Comment:
I authenticated the flags controller in the last PR so we don't need to include it in robots.txt